expo/gbe/models/remains.py

```diff
@@ -483,13 +483,29 @@ def append_alerts(self, alerts):
         '''
         return alerts
 
-    @property
-    def promo_small(self):
+    '''
```

Comment:
Docstring should be the first thing evaluated in the function - not preceding the function definition.